Cargo.toml

```
@@ -51,6 +51,11 @@ net2       = "0.2.33"
 all-features = true
 rustdoc-args = ["--cfg", "docsrs"]
 
+[package.metadata.playground]
+default-features = true
+features = ["os-poll", "os-util", "tcp", "udp", "uds"]
+all-features = false
```

How about just specifying the features needed?

```suggestion
[package.metadata.playground]
features = ["os-poll", "os-util", "tcp", "udp", "uds"]
```

`all-features` is `false` by default, so I don't think we need to explicitly set that.

Additionally, Mio has no default features, so `default-features = true` is not adding any features.